README.md

```diff
@@ -251,6 +228,22 @@ For example:
 package-registry --metrics-address 0.0.0.0:9000
 ```
 
+## Proxy Mode
+
+The Docker image of Package Registry is just an empty distribution without any packages.
+You can enable in Package Registry the proxy mode. This mode allows to take into account all the packages
+from other endpoint as part of the responses.
+
+This mode is enabled with the parameter `-feature-proxy-mode=true` (or `EPR_FEATURE_PROXY_MODE` environment variable).
+And it will use by default as proxy endpoint `https://epr.elastic.co`. This endpoint can be customized using the parameter `-proxy-to`
+(or `EPR_PROXY_TO`).
+For example:
+
+```
+package-registry --feature-proxy-mode=true -proxy-mode=https://epr.elastic.co
```

Comment:
```suggestion
package-registry --feature-proxy-mode=true -proxy-to=https://epr.elastic.co
```

Reply:
Ouch! I was checking the parameter spelling in the paragraph, and I missed it here 
Thanks!